package.json

```diff
@@ -100,7 +100,7 @@
     "ember-qunit": "^4.4.1",
     "ember-resolver": "^5.0.0",
     "ember-route-action-helper": "^2.0.7",
-    "ember-router-scroll": "1.1.0",
+    "ember-router-scroll": "DockYard/ember-router-scroll",
```

Comment:
Use a specific commit `DockYard/ember-router-scroll#695d894`

Reply:
@niranjan94 If we will use specific commits, Do we need to change it after every commit in the dependancy? Rather specifying Git Repo seems a pretty good option ?

Reply:
@kushthedude specifying commits lets us have control over the dependency. If you specify a repo+branch, then an accidental rogue commit on the dependency repo can and will break our builds. 

And yes. we will need to change it it periodically to a newer commit after verifying locally that the commits don't cause an issue. But this is only until the package publishes a version to npm.